Policies/KubernetesService/Azure_KubernetesService_AuthN_Enabled_AAD/Security - Kubernetes Service - EnableAADAuthentication.json

```diff
@@ -0,0 +1,66 @@
+{
+    "mode": "All",
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "The effect determines what happens when the policy rule is evaluated to match"
+        },
+        "allowedValues": [
+          "Audit",
+          "Disabled",
+          "Deny"
+        ],
+        "defaultValue": "Audit"
+      }
+    },
+    
+    "policyRule": {
+      "if": {
+        "allof":[
+          {
+            "field": "type",
+            "equals": "Microsoft.ContainerService/managedClusters"
+          },
+          {
+            "not":{
+              "anyof":[
+                {
+                  "allof":[
+                    {
+                      "field": "Microsoft.ContainerService/managedClusters/aadProfile.managed",
+                      "exists": "true"
+                    },
+                    {
+                      "field": "Microsoft.ContainerService/managedClusters/aadProfile.managed",
```

Comment:
Both conditions are exactly same is this intdended?

Reply:
Not same,
one checks if the field exists and other check if it equals to true or not.